src/components/BRB/BRBSchedules.tsx

```diff
@@ -228,6 +228,7 @@ const ArrowSmall = styled(Arrow)`
 `;
 
 const ParticipateButtonInnerText = styled(Span)`
+  font-family: Glancyr;
```

Comment:
Fallback font should be mentioned: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family